### DISCUSSION TITLE
I18N for Germany

### DISCUSSION THREAD

**Discussion by app-akrobaten** [DouyinFE/semi-design#90] [Ideas]
Hey do you think, you can also provide an i18 for date and time and label translations in german?

**Comment by shijiatongxue** [comment]
Hi there, we are glad to see you giving a pull request. See how to do it [here]( https://github.com/DouyinFE/semi-design/tree/main/packages/semi-ui/locale)!

**Comment by pointhalo** [comment]
In the latest v2.15.0-beta.0, LocaleProvider has added German support
![image](https://user-images.githubusercontent.com/88709023/178422913-7c448033-e566-4cfb-be57-c8a8412f0089.png)

**Comment by SudoUserReal** [comment]
![image](https://user-images.githubusercontent.com/29626873/183016482-1a1d3f74-84cf-49f3-807b-8db58190c7e8.png)
Support German, Italian and French [@jacob-lcs](https://github.com/jacob-lcs)  at 2.15.0-beta.0  @app-akrobaten
